bootstrap.d/x11-libs.yml

```diff
@@ -232,6 +232,72 @@ packages:
         environ:
           DESTDIR: '@THIS_COLLECT_DIR@'
 
+  - name: gtk+-3
+    architecture: '@OPTION:arch@'
+    source:
+      subdir: 'ports'
+      git: 'https://gitlab.gnome.org/GNOME/gtk.git'
+      tag: '3.24.23'
+      version: '3.24.23'
+    tools_required:
+      - system-gcc
```

Comment:
host-glib, wayland-scanner should be here too, otherwise it doesnt configure

Reply:
Thanks for noting it, I've added them to the list.